docs/ui/auth/fragments/angular/authenticator.md

```diff
@@ -61,6 +61,71 @@ Amplify UI Components use `slots` based off of the [Web Components slot element]
 | `totp-setup`           | Content placed inside of the totp-setup workflow for when a user opts to use TOTP MFA                                  |
 | `greetings`            | Content placed inside of the greetings navigation for when a user is signed in                                         |
 
+### Custom Form Fields
+
+If you'd like to customize the form fields in the Authenticator Sign In or Sign Up component you can do so by using the `formFields` property.
+
+The following is an example highlights the use of Authenticator with customized Sign Up form fields and [authentication with email](#authenticate-with-email-or-phone-number):
+
+*app.component.ts*
+```js
+import { Component } from '@angular/core';
+import { FormFieldTypes } from '@aws-amplify/ui-components';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  formFields: FormFieldTypes;
+  constructor() {
+    this.formFields = [
+      {
+        type: "email",
+        label: "Custom email Label",
+        placeholder: "custom email placeholder",
+        required: true,
+      },
+      {
+        type: "password",
+        label: "Custom Password Label",
+        placeholder: "custom password placeholder",
+        required: true,
+      },
+      {
+        type: "phone_number",
+        label: "Custom Phone Label",
+        placeholder: "custom Phone placeholder",
+        required: false,
+      },
+    ];
+  }
+}
+```
+
+*app.component.html*
+```html
+<amplify-authenticator usernameAlias="email">
+  <amplify-sign-up
+    slot="sign-up"
+    usernameAlias="email"
+    [formFields]="formFields"
+  ></amplify-sign-up>
+</amplify-authenticator>
+```
+
+Here is an example of the component in use:
+
+<docs-component-playground component-name="AuthenticatorWithSlots"></docs-component-playground>
+
+<amplify-callout warning>
+
+If you are using the `usernameAlias` prop with custom `slots` keep in mind that you must pass the `usernameAlias` prop value to both the Authenticator and custom slotted component since the slotted component overrides the configuration passed from the Authenticator.
```

Comment:
```suggestion
If you are using the `usernameAlias` prop with custom `slots`, keep in mind that you must pass the `usernameAlias` prop value to both the Authenticator and custom slotted component since the slotted component overrides the configuration passed from the Authenticator.
```